DEVELOP.md

```diff
@@ -73,6 +73,7 @@ gulp                    # Compile the JS and start the automatic
 To regenerate the `bundled.js` file without using the live reloader, run:
 
 ```
+gulp peg
```

Comment:
I typically just run `gulp prod` in lieu of more specific subcommands. Any reason not to just run that?